tools/static-assets/skel-full/imports/startup/both/useraccounts-configuration.js

```diff
@@ -0,0 +1,58 @@
+// The useraccounts package must be configured for both client and server to work properly.
```

Comment:
This app doesn't include this package, so this seems odd -- I think maybe you could just add a comment to the `imports/startup/both/index.js` detailing the kind of shared configuration you might have.